core/sql/ustat/hs_cli.cpp

```diff
@@ -524,12 +531,14 @@ Lng32 HSSample::create(NAString& tblName, NABoolean unpartitioned, NABoolean isP
            tableOptions = " WITH PARTITIONS";
         // If a transaction is running, the table needs to be created as audited.
         // Otherwise, create table as non-audited.
+        /* TEMPTEMP. Dave need to validate this change.
         if (TM->InTransaction())
           tableOptions += " ATTRIBUTE AUDIT";
         else
           tableOptions += " ATTRIBUTE NO AUDIT";
 
         tableOptions += getTempTablePartitionInfo(unpartitioned, isPersSample);
+        */
```

Comment:
I'm not sure why I moved that over -- I don't think it's related specifically to IUS. Maybe I mistook it for being part of one of your previous changes related to persistent samples.  Anyway, I tried making the change you suggested, and now there are test failures like this:

> *** ERROR[9214] Object TRAFODION.COMPGENERAL_TEST023.TRAF_SAMPLE_nn_nnnnnnnnnn_nnnnnn could not be created.
> 
> *** ERROR[15001] A syntax error occurred at or before:
> CREATE TABLE TRAFODION.COMPGENERAL_TEST023.TRAF_SAMPLE_nn_nnnnnnnnnn_nnnnnn LIK
> E TRAFODION.COMPGENERAL_TEST023.STEST WITH PARTITIONS EXTENT(8192, 16384) MAXEX
>  ^ (zzz characters from start of SQL statement)
> 
> *** ERROR[8822] The statement was not prepared.
> 

I think I'll leave this like it was, since it seems to be a separate issue and may require more investigation.